modules/sb-install-asb-operator.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+
+[id='sb-install-asb-operator-{context}']
+= Installing the {asb-name} Operator
+
+.Prerequisites
+
+* You have enabled service catalog.
+
+.Procedure
+
+The following procedure installs the {asb-name} Operator using the web console.
+
+. Create a project.
+.. Navigate in the web console to *Home* -> *Projects* and click *Create Project*.
+.. Enter `ansible-service-broker` in the *Name* field and click *Create*.
+. Create a cluster role binding.
+.. Navigate to *Administration* -> *Role Bindings* and click *Create Binding*.
+.. For the *Binding Type*, select *Cluster-wide Role Binding (ClusterRoleBinding)*.
+.. For the *Role Binding*, enter `ansible-service-broker` in the *Name* field.
+.. For the *Role*, select *admin*.
+.. For the *Subject*, choose the *Service Account* option, select the
+ `ansible-service-broker` namespace, and enter `ansible-service-broker` in the
+ *Subject Name* field.
+.. Click *Create*.
+. Navigate to the *Catalog* -> *Operator Hub* page. Verify that the `ansible-service-broker` project is selected.
+. Select *OpenShift Ansible Service Broker Operator*.
++
```

Comment:
This should be 'Automation Broker Operator'

Reply:
At present, the name of asb in operator hub is 'Automation Broker Operator'. 
@shawn-hurley Hope double confirm with you.

Reply:
As shawn-hurley said in other place, please ignore this comments at present. We will double confirm while downstream ready.